monitoring/perf_context_imp.h

```diff
@@ -9,6 +9,16 @@
 #include "util/stop_watch.h"
 
 namespace rocksdb {
+#if defined(NPERF_CONTEXT) || !defined(ROCKSDB_SUPPORT_THREAD_LOCAL)
+extern PerfContext perf_context;
+#else
+#if defined(OS_SOLARIS)
+extern __thread PerfContext perf_context_;
+#define perf_context (*get_perf_context());
+#else
```

Comment:
Add indentation for easier readability here. Otherwise it is easy to miss which if-block the second #else belongs to.

Reply:
This is the "make format" output. If I indent, "make format" will revert it back.

Reply:
Ah, ok. didn't realize that.